<launch>
  <!-- Map server -->
  <node name="goal_set" pkg="fcsc" type="goal_set" />

  <!-- Arguments -->
  <arg name="map_file" default="$(find fcsc)/maps/test5.yaml"/>
  <arg name="open_rviz" default="true"/>

  <!--rover-->
  <include file="$(find fcsc)/launch/rover.launch"/>

  <!-- Map server -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />

  <!-- AMCL -->
  <include file="$(find fcsc)/launch/rover_amcl.launch"/>

  <!-- move_base -->
  <include file="$(find fcsc)/launch/move_base_fcsc.launch"/>

  <!-- rviz -->
  <node pkg="rviz" type="rviz" args="-d '$(find mecanumrover_samples)/configuration_files/rviz/navigation.rviz'" name="visualizer" respawn="true" />
</launch>
